droidlet/tools/artifact_scripts/tracked_checksums/datasets.txt

```diff
@@ -1 +1 @@
-3e7dfe1694860f8339e4ed15ae3e5a491ece2a90
+8559f0495013cd9b4a54f5e86db8b1d3c1f03128
```

Comment:
what's changed in the datasets in this PR?

Reply:
some of the hi-pri commands